src/components/icon/Icon.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+export type COLOURS = 'neutral' | 'mint' | 'violet' | 'ruby' | 'gold' | 'aqua' | 'teal';
+export type TINTS = 'lightest' | 'light' | 'normal' | 'dark' | 'darkest';
+
+interface IconProps {
```

Comment:
add behind `IconProps` this: `extends Omit<BoxProps , {{props you are overriding}}>`

This is for `...others` spread that is used on line 28